assets/dev/scss/frontend/_global.scss

```diff
@@ -2,6 +2,10 @@
 // Global selectors for Elementor Front-End
 //
 
+html {
```

Comment:
Can we include this styling to every page? 
@rami-elementor

Reply:
I think that we can because this is was the default behaviour in previous version. The change doesn't break backwards compatibility.

@louiswol94 what do you think? Does it sounds correct?

Reply:
@hein-obox another question, does it affect the Elementor Editor or the WordPress Admin?

Reply:
I mean, does it affect compare to previous Elementor versions?

Reply:
@rami-elementor 
Good question.

Elementor Editor: could be affected, although generally speaking the anchor links are disabled in the editor.
WordPress Admin: not affected.

Could I move this code to another file? 
I could create a file like `_general.scss` or `_smooth-scroll.scss` that is always enqueued on the frontend.

What do you think?

Reply:
> I mean, does it affect compare to previous Elementor versions?

The previous versions only had support for the frontend.

Reply:
If you can make sure the change affect only the frontend, you can leave it as it is. Otherwise, the `_smooth-scroll.scss` file imported to `frontend.scss` is a better approach.

Reply:
@rami-elementor 

> Otherwise, the `_smooth-scroll.scss` file imported to `frontend.scss `is a better approach.

Implemented like this.